inference/engine.go

```diff
@@ -39,6 +39,10 @@ type Engine struct {
 	// converted to diagnostics and returned along with the current inferred map whenever users
 	// request it.
 	conflicts *conflictGrouping
+	// controlledTriggersBySite stores the set of controlled triggers for each site if the site is
+	// controls any triggers. This field is for internal use in the struct only and should not be
```

Comment:
Can't parse the end of this first sentence: "if the site is controls any triggers". Should it be "if the site controls any triggers"?

Reply:
Good catch.